node/src/chain_spec.rs

```diff
@@ -251,6 +257,10 @@ pub fn integritee_solo_fresh_config() -> Result<ChainSpec, String> {
 		None,
 		// Protocol ID
 		Some("teer"),
+		// Arbitrary string. Nodes will only synchronize with other nodes that have the same value
+		// in their `fork_id`. This can be used in order to segregate nodes in cases when multiple
+		// chains have the same genesis hash.
+		None,
```

Comment:
if this is needed now, I suggest to add this to the hardcoded specs in the `res` folder too. We had trouble in the past when hardcoded specs don't comply with what is generated with `-fresh`

Reply:
updated hardcoded specs with:
    -  `./target/release/integritee-node build-spec --chain cranny --raw > node/res/cranny.json`
    - `./target/release/integritee-node build-spec --chain integritee-solo --raw > node/res/integritee-solo.json`

Reply:
There was a mistake in updating the hardcoded-specs; You must not change the genesis-field. Otherwise, you can't connect to the running integritee-network. You either just add the field manually, or you recreate the chain-spec and migrate old fields you need to keep, see [encointer-parachain](https://github.com/encointer/encointer-parachain/blob/master/scripts/update_hardcoded_chain_specs.py). Note, the script documentation does no longer explain the correct cli args.

However, as you could just run the node and connect to the running network, and as fork-id is an optional field, I think you don't need to update the chain-spec at all in this case.

Reply:
Yes, the chain is able to sync with the old chain spec. Should I still update (with the help of the script this time)?

Reply:
I don't think so.

Reply:
absolutely not!
Genesis may not be touched! I was talking about addin the one field to the json which is new

Reply:
I added the scripts from the encointer parachain (thanks @clangenb ) and run them for the two specs. However, nothing was updated. Is there a possibility that the chain-specs don't include `None` values?

Either way: The chain can be run with the original chain-spec and it syncs.

Reply:
ok, we can leave it then